docs/docsite/rst/porting_guide_2.3.rst

```diff
@@ -0,0 +1,193 @@
+.. _porting_2.1_guide:
+
+*************************
+Ansible 2.3 Porting Guide
+*************************
+
+This section discusses the changes to how you use Ansible between Ansible 2.2 and Ansible 2.3.
+
+It is intended to assist in updating your playbooks, plugins, etc so they will work with this version of Ansible.
+
+
+We suggest you read this page along with https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#23-ramble-on---2017-04-12 before upgrading Ansible to understand what, if any updates you will need to make.
+
+This document is part of a collection on porting, the rest can be found at :ref:`porting_guides <Porting Guides>`.
+
+.. contents:: Topics
+
+Playbook
+========
+
+
+Deprecated
+==========
+
+
+
+Use of multiple tags
+--------------------
+
+Specifying ``--tags`` (or ``--skip-tags``) multiple times on the command line currently leads to the last one overriding all the previous ones. This behaviour is deprecated. In the future, if you specify --tags multiple times the tags will be merged together. From now on, using ``--tags`` multiple times on one command line will emit a deprecation warning. Setting the ``merge_multiple_cli_tags`` option to True in the ``ansible.cfg`` file will enable the new behaviour.
```

Comment:
Please change to "...currently leads to the last tag specified overriding all of the previously specified tags."